src/js/controller/LayersListController.js

```diff
@@ -113,8 +115,7 @@
     if (el.classList.contains('button')) {
       this.onButtonClick_(el);
     } else if (el.classList.contains('layer-name')) {
-      index = pskl.utils.Dom.getData(el, 'layerIndex');
-      this.piskelController.setCurrentLayerIndex(parseInt(index, 10));
+      this.focusLayerElement_(el.parentNode);
```

Comment:
I guess you had conflicts between the click and doubleClick handlers, which is why you had to remove the RESET event when setting the layer index.

What was the issue exactly ? Was the second click event fired after the dblclick event, causing the markup to be regenerated ?